Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM openjdk:11-jre-slim
 LABEL maintainer="Andrew Gaul <andrew@gaul.org>"
 
+RUN mkdir /data ;\
+    useradd -rm -d /opt/s3proxy -s /bin/sh -u 10001 -U javauser
```

Comment:
as a variable it would not do much good this is build time only I don't see a reason we would change the user in docker then. I also have not seen that in other container solutions. When running the container docker/k8s you can't pass a user ID only a UID. The purpose of this line is to have a uid that is known